config/config.toml.example

```diff
@@ -60,6 +60,9 @@ enable-table-lock = false
 # delay-clean-table-lock is used to control the time (Milliseconds) of delay before unlock the table in the abnormal situation.
 delay-clean-table-lock = 0
 
+# Maximum split regions numbers in split region syntax.
```

Comment:
Maximum number of the splitting region, which is used by the split table statement

Reply:
done.